warehouse/accounts/views.py

```diff
@@ -32,6 +32,7 @@
 @view_config(
     route_name="accounts.profile",
     renderer="accounts/profile.html",
+    uses_session=True,
```

Comment:
Hmm, I'd prefer not to set ``uses_session`` on the profile, that will make it hard to cache the profile view (effectively impossible for logged in users). Let me think on this a few.